sycl/source/detail/device_image_impl.hpp

```diff
@@ -1053,7 +1053,8 @@ class device_image_impl {
       auto NewSupportingDevsEnd = std::remove_if(
           SupportingDevs.begin(), SupportingDevs.end(),
```

Comment:
Does that have to be a vector of `sycl::device`s?

Reply:
No, but changing it to a vector of `device_impl` would require changing the function signature and all its dependents, and that won't be easy.